app/models/host/base.rb

```diff
@@ -214,17 +214,20 @@ def set_comment(parser)
       desc 'Note that available facts depend on what facts have been uploaded to Foreman,
            typical sources are Puppet facter, subscription manager etc.
            The facts can be out of date, this macro only provides access to the value stored in the database.'
+      optional :fact_names, Array, desc: 'A list of fact names to return. If empty all facts are returned'
```

Comment:
@ofedoren is this also correct documentation for `*args` parameters, or is there a specific tag for that?

Reply:
Agh, my bad for not noticing this earlier :/

Well, it's not as wrong. It's still quite optional :) But for `*args` params, I'd use one of `list`/`splat`/`rest` instead of `optional`.

Reply:
Sure: https://github.com/theforeman/foreman/pull/9850